modules/industry/src/chemicals_industry.py

```diff
@@ -0,0 +1,198 @@
+import eurocalliopelib.utils as ec_utils
+import pandas as pd
+from utils import formatting
+from utils import jrc_idees_parser as jrc
+
+H2_LHV_KTOE = 2.863  # 0.0333 TWh/kt LHV -> 2.863ktoe/kt
+MWH_PER_T_TO_KTOE_PER_KT = 0.08598  # 1 MWh/t -> 1 GWh/kt -> 0.08598 ktoe/kt
+METHANOL_LHV_KTOE = 0.476  # 19.915 MJ/kg LHV -> 19915000MJ/kt -> 0.476ktoe/kt
+
+CAT_NAME_CHEM = "Chemicals Industry"
+
+
+def get_chem_demand_df(
+    year_range: list,
+    path_energy_balances: str,
+    path_cat_names: str,
+    path_carrier_names: str,
+    path_jrc_energy: str,
+    path_jrc_production: str,
+    path_output: str = "",
```

Comment:
```suggestion
    path_output: Optional[str] = None,
```
Then `if path_output is not None:` later on